text/0000-issue-workflow.md

```diff
@@ -0,0 +1,222 @@
+
+A workflow template for taking issues from creating to resolution. Project repositories would opt-in to using this
+workflow and reference it.
+
+# Motivation
+[motivation]: #motivation
+
+As a community member, I would like to get reasonable response times.
+
+As a community member, I would like to see what state issues are currently in.
+
+As a contributor, I would like to easily identify what issues are ready to be worked on.
+
+# What it is
+[what-it-is]: #what-it-is
+
+The target audience for this RFC is project contributors. It defines a workflow that project contributors and maintainers
+should follow to provide the level of community support that is desired.
+
+The outcome of this RFC would be the documentation of this workflow being published in the
+[community](https://github.com/buildpacks/community) repo.
+
+### Labels
+
+##### Statuses
+- status/triage
+- status/requires-rfc
+- status/duplicate
+- status/invalid 
+- status/incomplete 
+- status/blocked
+- status/discussion-needed
+- status/not-planned
+- status/ready
+- status/in-progress
+- status/resolved
+- status/code-approved
+- status/user-accepted
+
+##### Types
+- type/support
+- type/enhancement
+- type/bug
+- type/research
+- type/chore
+
+# How it Works
+[how-it-works]: #how-it-works
+
+This is the technical portion of the RFC, where you explain the design in sufficient detail.
+
+### Standard Workflow
+
+```text
+     +----------+  +----------------------+            +----------+  +----------------------+
+     |             |                      |            |             |                      |
+     |             |     status/triage    |            |             |    status/triage     |
+     |             |                      |         P  |             |                      |
+     |             +----------------------+         U  |             +----------------------+
+     |                         |                    L  |                         |
+  I  |                         |                    L  |                         |
+  S  |             +-----------v----------+            |             +-----------v----------+
+  S  |             |                      |         R  |             |                      |
+  U  |             |     status/ready     |         E  |             | status/code-approved |
+  E  |             |                      |         Q  |             |                      |
+  S  |             +----------------------+         U  |             +----------------------+
+     |                         |                    E  |                         |
+     |                         |                    S  |                         |
+     |             +-----------v----------+         T  |             +-----------v----------+
+     |             |                      |         S  |             |                      |
+     |             |  status/in-progress  |            |             | status/user-accepted |
```

Comment:
Non-project contributors cannot set labels or assign issues to themselves. We might want to rethink or add additional details here with that in mind.